applications/ContactStructuralMechanicsApplication/custom_processes/alm_fast_init_process.cpp

```diff
@@ -32,21 +32,12 @@ namespace Kratos
         // We initialize the penalty parameter
         const double& epsilon = mrThisModelPart.GetProcessInfo()[INITIAL_PENALTY];
         
-        bool init_delta_normal = false;
-        Matrix zero_delta_normal;
-        if (mrThisModelPart.GetProcessInfo()[CONSIDER_NORMAL_VARIATION] == true)
-        {
-            init_delta_normal = true;
-            const unsigned int dimension = mrThisModelPart.GetProcessInfo()[DOMAIN_SIZE];
-            zero_delta_normal = ZeroMatrix( dimension, dimension );
-        }
-        
         // We iterate over the node
         NodesArrayType& nodes_array = mrThisModelPart.Nodes();
         const int num_nodes = static_cast<int>(nodes_array.size());
         
         #pragma omp parallel for firstprivate(zero_vector)
```

Comment:
take care, ZeroVector is an expression, not a vector. i would use the ZeroVector inside instead of allocating an object

Reply:
OK